acra/security-controls/transport-security/separate_encryption.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Separate encryption [ENTERPRISE]
+bookCollapseSection: true
+weight: 3
+---
+
+# Separate encryption [ENTERPRISE]
```

Comment:
actually, it's not an enterprise version... it's available for everyone who uses acrawriter on the app side. and acrawriter is open-sourced.